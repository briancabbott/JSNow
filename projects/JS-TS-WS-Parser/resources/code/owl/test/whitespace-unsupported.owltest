y
---#using owl.v2
.whitespace 'x'
a = 'y'
