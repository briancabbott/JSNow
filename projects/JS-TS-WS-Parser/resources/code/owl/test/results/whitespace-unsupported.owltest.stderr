error: specifying whitespace is unsupported in versions before owl.v3

  #using owl.v2
         ~~~~~~
  .whitespace 'x'
  ~~~~~~~~~~~~~~~

